src/features/world/scenes/Kingdom.ts

```diff
@@ -18,11 +66,100 @@ export class KingdomScene extends BaseScene {
     super.preload();
   }
 
+  setUpKingdomNPCS() {
+    KINGDOM_NPCS.forEach(({ npc, x, y, direction = "right", faction }) => {
+      const container = new BumpkinContainer({
+        scene: this,
+        x,
+        y,
+        clothing: {
+          ...NPC_WEARABLES[npc],
+          updatedAt: 0,
+        },
+        direction,
+      });
+
+      container.setDepth(y);
+      (container.body as Phaser.Physics.Arcade.Body)
+        .setSize(16, 20)
+        .setOffset(0, 0)
+        .setImmovable(true)
+        .setCollideWorldBounds(true);
+
+      this.physics.world.enable(container);
+      this.colliders?.add(container);
+      this.triggerColliders?.add(container);
+
+      switch (faction) {
+        case "bumpkins":
+          this.bumpkinsRecruiterFactionNPC = container;
+          break;
+        case "goblins":
+          this.goblinsRecruiterFactionNPC = container;
+          break;
+        case "sunflorians":
+          this.sunfloriansRecruiterFactionNPC = container;
+          break;
+        case "nightshades":
+          this.nightshadesRecruiterFactionNPC = container;
+          break;
+      }
+    });
+  }
+
+  makeAllFactionRecruiterNPCsInteractive() {
```

Comment:
It appears we can click the NPCs from anywhere on the map. Other areas in the game you need to be close to them to have a Dialogue.

Are you able to reuse the `this.initialiseNPCs()` instead of registering custom handlers?